README.md

```diff
@@ -1,4 +1,4 @@
-# Slugify
+# Slugify :abcd:
```

Comment:
To have the emoji working outside of GitHub, maybe we should change it to 🔤 - what do you think?

Reply:
:abc: is also good for me, I will update immediately!